.gitignore

```diff
@@ -184,6 +185,7 @@ replay_pid*
 .idea/**/usage.statistics.xml
 .idea/**/dictionaries
 .idea/**/shelf
+.vscode/settings.json
```

Comment:
I think, this and your change in `.vscode/settings.json` is OK. Future contributors not following our guidelines to use IntelliJ will benefit (because they won't do commits changing `.vscode/settings.json`).